R/robustbase-lmrob-tidiers.R

```diff
@@ -0,0 +1,86 @@
+#' @templateVar class lmrob
+#' @template title_desc_tidy_lm_wrapper
```

Comment:
Similar feedback here in terms of implementing all new methods being much better than wrapping the `*.lm()` tidiers.